docs/codeql/codeql-language-guides/basic-query-for-cpp-code.rst

```diff
@@ -52,7 +52,7 @@ After the initial ``import`` statement, this simple query comprises three parts
 +---------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
 | Query part                                                    | Purpose                                                                                                           | Details                                                                                                                |
 +===============================================================+===================================================================================================================+========================================================================================================================+
-| ``import cpp``                                                | Imports the standard CodeQL libraries for C/C++.                                                                  | Every query begins with one or more ``import`` statements.                                                             |
+| ``import c-cpp``                                                | Imports the standard CodeQL libraries for C/C++.                                                                  | Every query begins with one or more ``import`` statements.                                                             |
```

Comment:
Here and elsewhere: do not change the imports, as we have not renamed the standard CodeQL libraries, so the renamed imports will not work.